### ISSUE TITLE
Failing Sannysoft Bot Test on Headless Chromium

### ISSUE THREAD

**Issue by shubhamshah02** [Granitosaurus/playwright-stealth#4]
Hey there, the tests in this repo are outdated. This plugin is resulting in red in "Plugins Length", "Plugins is of type PluginArray", "WebGL Vendor" and "WebGL Renderer" in headless mode. 

Is this possible to fix?

**Comment by AhilanAshwin** [comment]
Yes, i'm having the same issue. Specifically, the "Plugins Length" and "Plugins is of type PluginArray". Not really familiar with JS. Would love for a quick fix or how to debug.
